frontend_app_file/src/component/Kanban.jsx

```diff
@@ -0,0 +1,386 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import { v4 as uuidv4 } from 'uuid'
+import Board from '@lourenci/react-kanban'
+import '@lourenci/react-kanban/dist/styles.css'
+import {
+  IconButton,
+  PromptMessage,
+  handleFetchResult,
+  putFileContent,
+  removeLocalStorageItem,
+  getLocalStorageItem,
+  setLocalStorageItem
+} from 'tracim_frontend_lib'
+
+require('./Kanban.styl')
+
+function moveElem (array, fromPosition, toPosition, elem) {
```

Comment:
I was about to advice using a lodash utility for that but it appears lodash doesn't have such a function.

So just don't forget to write a few unit tests for it. If you took it from stack, we usually add a comment with the source link

Reply:
I wrote this myself, good point for the unit testing of this function.